### DISCUSSION TITLE
HybridizationType: UNSPECIFIED and OTHER

### DISCUSSION THREAD

**Discussion by jack-cadd** [rdkit/rdkit#5035] [Q&A] [answered]
As I look at the hybrid types provided by RDKit, UNSPECIFIED and OTHER exist, I'm confused about the differences between the two types, and under what circumstances UNSPECIFIED appears.

**Comment by greglandrum** [comment, marked as answer]
@jack-cadd, UNSPECIFIED is the default value; it indicates that hybridization has not been calculated/assigned (for example if a molecule hasn't been sanitized). OTHER can be assigned if the RDKit does not recognize the coordination environment of the atom.

**Reply by jack-cadd** [reply]
Thank you for your explanation, now I understand the UNSPECIFIED. As for OTHER, my original understanding was that the other hybrid types besides S, SP, SP2, SP3, SP3D, SP3D2. Is this the same as your explanation?

**Reply by greglandrum** [reply]
Yes, OTHER can be assigned if the hybridization is something other than S, SP, SP2, etc.

**Reply by jack-cadd** [reply]
Got it, thank you^-^
